tests/puppeteer/pages/page.js

```diff
@@ -0,0 +1,14 @@
+module.exports = class Page {
+
+  constructor(page) {
+  }
+
+  async getPageTitle() {
+    return global.page.title();
```

Comment:
Using global is a bad practice, use dependency injection instead.
```js
module.exports = class Page {

  constructor(page) {
      this.page = page;
  }

  async getPageTitle() {
    return this.page.title();
   }

  async goTo(URL) {
    await this.page.goto(URL);
  }
};
```

It will be easier to maintain

Reply:
We kinda have a problem with this implementation. I'd love to use DI of course, but our `page` object is created outside, in `helpers.js` which is automatically pulled by Mocha. AFAIK we can't get it from there in our main script.

Reply:
We changed the way we handle the `page` object. Probably better now.